app/forms/SchemeDetails/SchemeDetailsFormProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.SchemeDetails
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import play.api.data.Form
+
+class SchemeDetailsFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[Option[String]] =
+    Form(
+      "value" -> optionalText()
+        .verifying(maxLength(150, "schemeDetails.error.length"))
```

Comment:
Are we sure that there is a max length constraint here ?
It should be, but I couldn't verify that limit is 150 chars.

Reply:
@cyeniceri So the ticket says there shouldn't be any validation on this page however this doesn't seem right. Can remove the validation if needed however surely at the least a maxLength should be enforced?

Reply:
@emrawsy I agree with you, there should be a limit for both fields, we shouldn't assume it. 
Can you ask this to Sanjay and update the ticket as well for both fields ?

Reply:
@cyeniceri sure, will do